examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/PaywallTesterApp.kt

```diff
@@ -1,9 +1,45 @@
 package com.revenuecat.paywallstester
 
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.navigation.NavHostController
+import androidx.navigation.NavType
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import androidx.navigation.navArgument
+import com.revenuecat.paywallstester.ui.screens.AppScreen
 import com.revenuecat.paywallstester.ui.screens.main.MainScreen
+import com.revenuecat.paywallstester.ui.screens.paywall.PaywallScreen
 
 @Composable
-fun PaywallTesterApp() {
-    MainScreen()
+fun PaywallTesterApp(
+    navController: NavHostController = rememberNavController(),
+) {
+    AppNavHost(navController = navController)
+}
+
+@Composable
+private fun AppNavHost(
+    navController: NavHostController,
+    modifier: Modifier = Modifier,
+) {
+    NavHost(
+        navController,
+        startDestination = AppScreen.Main.route,
+        modifier = modifier,
+    ) {
+        composable(AppScreen.Main.route) {
+            MainScreen(navigateToPaywallScreen = { offering ->
+                navController.navigate(AppScreen.Paywall.route.plus("/${offering?.identifier}"))
```

Comment:
I hate this way of passing parameters... Right now, we are also only passing the offering id, not the entire offering, since it's not parcelable... But we can iterate on this later... This is the way it's done in the official documentation: https://developer.android.com/jetpack/compose/navigation

Reply:
Relying on offerings cache is probably fine